File: newDemo/newDemo/Base/Item/BaseCollectionViewCell.m
//
//  BaseCollectionViewCell.m
//  BaseToolSystem
//
//  Created by 靳建南 on 2018/11/6.
//  Copyright © 2018年 靳建南. All rights reserved.
//

#import "BaseCollectionViewCell.h"

@implementation BaseCollectionViewCell

-(id)initWithFrame:(CGRect)frame{
    
    if (self = [super initWithFrame:frame]) {
        self.contentView.backgroundColor = [UIColor whiteColor];
    }
    
    return self;
}

@end

File: BaseToolClass/BaseToolClass/Base(基类)/Main/BaseCollectionViewCell.swift
//
//  BaseCollectionViewCell.swift
//  BaseToolClass
//
//  Created by 靳建南 on 2018/11/9.
//  Copyright © 2018年 靳建南. All rights reserved.
//

import UIKit

class BaseCollectionViewCell: UICollectionViewCell {
    
    var FrameW:CGFloat!
    var FrameH:CGFloat!
    
    override init(frame: CGRect) {
        super.init(frame: frame)
        
        self.FrameW = self.frame.width
        self.FrameH = self.frame.height
    }
    
    required init?(coder aDecoder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
}
